app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -111,7 +114,7 @@ public DeleteHelper(NotificationHelper notificationHelper,
                     throw new RuntimeException("Failed to nominate for deletion");
                 }).flatMap(result -> {
                     if (result) {
-                        return pageEditClient.appendEdit("User_Talk:" + username, userPageString + "\n", summary);
+                        return pageEditClient.appendEdit("User_Talk:" + creatorName, userPageString + "\n", summary);
```

Comment:
If the creator name is null, I think it is better to raise an exception as 
```throw new RuntimeException("Failed to nominate for deletion");``` instead of appending edits to the page of "https://commons.wikimedia.org/wiki/User_talk:page does not exist". So please raise the exception instead of filling the null string with "page not exist"